src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -81,38 +104,30 @@ class IOUAmountPage extends React.Component {
                         {this.props.selectedCurrency}
                     </Text>
                     {this.props.isSmallScreenWidth
-                        ? <Text style={styles.iouAmountText}>{this.props.amount}</Text>
+                        ? <Text style={styles.iouAmountText}>{this.state.amount}</Text>
                         : (
-                            <View>
-                                <TextInputFocusable
-                                        style={[styles.iouAmountTextInput,
-                                            {width: Math.max(5, this.state.textInputWidth)}]}
-                                        onKeyPress={(event) => {
-                                            this.props.numberPressed(event.key);
-                                            event.preventDefault();
-                                        }}
-                                        ref={el => this.textInput = el}
-                                        defaultValue={this.props.amount}
-                                        textAlign="left"
-                                />
-                                <Text
-                                    style={[styles.iouAmountText, styles.invisible, {left: 100000}]}
-                                    onLayout={e => this.setState({textInputWidth: e.nativeEvent.layout.width})}
-                                >
-                                    {this.props.amount}
-                                </Text>
-                            </View>
+                            <TextInputAutoGrow
+                                    inputStyle={styles.iouAmountTextInput}
+                                    textStyle={styles.iouAmountText}
+                                    onKeyPress={(event) => {
+                                        this.updateAmountIfValidInput(event.key);
+                                        event.preventDefault();
+                                    }}
+                                    ref={el => this.textInput = el}
+                                    value={this.state.amount}
+                                    textAlign="left"
```

Comment:
I don't see `textAlign` defined in the props. Am I just missing it somewhere? Does this need to be in props at all or is it always going to be "left"?

Reply:
I guess I forgot to remove this prop.